src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div>
+    <div>
+      <h4>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></h4>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-owns="id"
+      role="combobox"
+      tabindex="0"
+      v-model="selectLabel"
+    >
```

Comment:
Totally personal preference, but do we like having these on the next line? I always put the on the last line

Reply:
I like putting them on the last line because I don't like the way inside elements start lining up with all the attributes above it.

Reply:
this is just habit for me,  I don't really care either way